src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -29,8 +29,14 @@ public class Command
 
         private bool _running = false;
 
-        private Command(string executable, string args)
+        //the command is not loaded from nuget libraries or from AppContext.BaseDirectory. It is machine specific.
+        //see https://github.com/dotnet/cli/issues/382
+        private readonly bool _isUnsafe;
+
+        private Command(string executable, string args, bool isUnsafe)
         {
+            _isUnsafe = isUnsafe;
```

Comment:
In this context, isUnsafe seems like it really means isLoadedFromPath.

Are there any other preconditions Command would be expected to check for?